draft-ietf-quic-tls.md

```diff
@@ -651,6 +651,17 @@ verification that the identity of the server is included in a certificate and
 that the certificate is issued by a trusted entity (see for example
 {{?RFC2818}}).
 
+Note:
+
+: Where servers provide certificates for authentication, the size of
+  the certificate chain can consume a large number of bytes.  Controlling the
+  size of certificate chains is critical to performance in QUIC as servers are
+  limited to sending 3 bytes for every byte received prior to validating the
+  client address; see Section 8.1 of {{QUIC-TRANSPORT}}.  The size of a
+  certificate chain can managed by limiting the number of names or extensions;
+  using keys with small public key representations, like ECDSA; or, by using
```

Comment:
s/or/and/?

Reply:
I think that a logical OR is sufficient here.  Though nothing stops someone from doing more than one, and maybe there are things that mean you need to (like being unwilling to reduce the number of cnames on certificates for $reasons.